src/routes/users/updateUser.ts

```diff
@@ -0,0 +1,101 @@
+import {
+  RawReplyDefaultExpression,
+  RawRequestDefaultExpression,
+  RawServerDefault,
+  RouteHandler,
+  RouteOptions,
+} from 'fastify';
+import UserSchema from '../../schemas/User.json';
+import { UserRequestParams } from '../../types/UserRequestParams';
+import UserRequestParamsSchema from '../../schemas/UserRequestParams.json';
+import { User } from '../../types/User';
+import UserModel from '../../models/user';
+import { ErrorSchema } from '../../types/ErrorSchema';
+import ErrorSchemaJson from '../../schemas/ErrorSchema.json';
+import { capitalize } from '../../utils/capitalize';
+type Reply = User | { error: ErrorSchema };
+type UpdateUserRoute = {
+  Body: User;
+  Params: UserRequestParams;
+  Reply: Reply;
+};
+
+const url = '/user/:id';
+
+const handler: RouteHandler<UpdateUserRoute> = async (req, reply) => {
+  try {
+    const { id: userId } = req.params;
+    const { firstName, areaId, lastName } = req.body;
+
+    const newUserFirstName = capitalize(firstName);
```

Comment:
We had agreed to move this directly to the object where we are doing the insertion, please apply that change.